src/m_misc.c

```diff
@@ -1159,11 +1159,11 @@ default_t defaults[] = {
   },
 
   {
-    "input_mouselook",
+    "input_freelook",
     NULL, NULL,
     {0}, {UL,UL}, input, ss_keys, wad_no,
-    "key to toggle mouselook",
-    input_mouselook, { {0, 0} }
+    "key to toggle free look",
+    input_freelook, { {0, 0} }
```

Comment:
Resets the old binding. Too annoying?

Reply:
We could just mention it in the changelog, no problem.